<HTML>
<HEAD>
<TITLE>Topic Report: J/qtl Help</TITLE>
<LINK REL=stylesheet TYPE=text/css HREF=report.css>
</HEAD><BODY>
<H1>Topic Report: J/qtl Help</H1><p>&nbsp;</p>
<i>To print this report, right click and choose "Print" from the pop-up context menu.</i><p>
<p><i> You can use the Contents, Topics, and Index tabs on the left to navigate within the report.</i>
<p><i> This HTML file and also a tab-delimited text file are available in the \report subdirectory below your project directory.</i>
<p>&nbsp;</p><p><b>Report Date/Time:</b> 11/21/2008 2:05:04 PM
<p><b>Total Number of Topics: </b> 145
<p><b>Number of Index Entries: </b> 0
<p>&nbsp;</p>
<a name="Calculate_Genotype_Probabilities.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Calculate Genotype Probabilities</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Calculate_Genotype_Probabilities</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Calculate_Genotype_Probabilities.htm" target=_blank>Calculate_Genotype_Probabilities.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="License.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Copyright and License</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Copyright_and_License</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\License.htm" target=_blank>License.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Display_and_Estimate_Genetic_Map.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Display and Estimate Genetic Map</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Display_and_Estimate_Genetic_Map</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Display_and_Estimate_Genetic_Map.htm" target=_blank>Display_and_Estimate_Genetic_Map.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\Edit_Preferences.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Edit Preferences</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Edit_Preferences</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\Edit_Preferences.htm" target=_blank>HTML\Edit_Preferences.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\Edit_QTL_Basket.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Edit QTL Basket</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Edit_QTL_Basket</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\Edit_QTL_Basket.htm" target=_blank>HTML\Edit_QTL_Basket.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Effect_Plot.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Effect Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Effect_Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Effect_Plot.htm" target=_blank>Effect_Plot.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\FirstTopic.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> FirstTopic</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> FirstTopic</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\FirstTopic.htm" target=_blank>HTML\FirstTopic.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Fit_QTL.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Fit QTL</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Fit_QTL</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Fit_QTL.htm" target=_blank>Fit_QTL.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Fit_Results.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Fit Results</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Fit_Results</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Fit_Results.htm" target=_blank>Fit_Results.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\Genome_Scans.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Genome Scans</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Genome_Scans</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\Genome_Scans.htm" target=_blank>HTML\Genome_Scans.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Genotype_Plot.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Genotype Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Genotype_Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Genotype_Plot.htm" target=_blank>Genotype_Plot.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Graph_Settings.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Graph Settings</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Graph_Settings</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Graph_Settings.htm" target=_blank>Graph_Settings.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\Graphs.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Graphs</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Graphs</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\Graphs.htm" target=_blank>HTML\Graphs.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Histogram_and_Scatterplot.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Histogram and Scatterplot</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Histogram_and_Scatterplot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Histogram_and_Scatterplot.htm" target=_blank>Histogram_and_Scatterplot.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\Installation_and_Uninstallation.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Installation and Uninstallation</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Installation_and_Uninstallation</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\Installation_and_Uninstallation.htm" target=_blank>HTML\Installation_and_Uninstallation.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Jittermap.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Jittermap</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Jittermap</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Jittermap.htm" target=_blank>Jittermap.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\Load_Cross_Data.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Load Cross Data</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Load_Cross_Data</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\Load_Cross_Data.htm" target=_blank>HTML\Load_Cross_Data.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\One_QTL_Genome_Scan.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> One QTL Genome Scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> One_QTL_Genome_Scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\One_QTL_Genome_Scan.htm" target=_blank>HTML\One_QTL_Genome_Scan.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\One_QTL_Genome_Scan_Summary.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> One QTL Genome Scan Summary</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> One_QTL_Genome_Scan_Summary</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\One_QTL_Genome_Scan_Summary.htm" target=_blank>HTML\One_QTL_Genome_Scan_Summary.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\One_QTL_Scan_Plot.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> One QTL Scan Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> One_QTL_Scan_Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\One_QTL_Scan_Plot.htm" target=_blank>HTML\One_QTL_Scan_Plot.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Scan_One_Graph_Settings.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> One QTL Scan Plot Setting</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> One_QTL_Scan_Plot_Settings</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Scan_One_Graph_Settings.htm" target=_blank>Scan_One_Graph_Settings.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\R_qtl_Function_Reference.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R/qtl Function Reference</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> R_qtl_Function_Reference</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\R_qtl_Function_Reference.htm" target=_blank>HTML\R_qtl_Function_Reference.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\addint.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Add pairwise interaction to a multiple-QTL model</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtladdint</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\addint.html" target=_blank>HTML\addint.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\add.threshold.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Add significance threshold to plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtladd_threshold</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\add.threshold.html" target=_blank>HTML\add.threshold.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\addtoqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Add to a qtl object</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtladdtoqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\addtoqtl.html" target=_blank>HTML\addtoqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\map10.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: An example genetic map</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlmap10</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\map10.html" target=_blank>HTML\map10.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\badorder.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: An intercross with misplaced markers</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlbadorder</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\badorder.html" target=_blank>HTML\badorder.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\arithscanperm.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Arithmetic Operators for permutation results</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlarithscanperm</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\arithscanperm.html" target=_blank>HTML\arithscanperm.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\arithscan.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Arithmetic Operators for scanone and scantwo results</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlarithscan</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\arithscan.html" target=_blank>HTML\arithscan.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\bayesint.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Bayesian credible interval</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlbayesint</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\bayesint.html" target=_blank>HTML\bayesint.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.scanoneboot.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Bootstrap confidence interval for QTL location</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_scanoneboot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.scanoneboot.html" target=_blank>HTML\summary.scanoneboot.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\scanoneboot.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Bootstrap to get interval estimate of QTL location</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlscanoneboot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\scanoneboot.html" target=_blank>HTML\scanoneboot.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\calc.genoprob.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Calculate conditional genotype probabilities</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcalc_genoprob</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\calc.genoprob.html" target=_blank>HTML\calc.genoprob.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\chrlen.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Chromosome lengths in QTL experiment</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlchrlen</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\chrlen.html" target=_blank>HTML\chrlen.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\clean.scantwo.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Clean up scantwo output</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlclean_scantwo</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\clean.scantwo.html" target=_blank>HTML\clean.scantwo.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\cbind.scanoneperm.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Combine columns from multiple scanone permutation results</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcbind_scanoneperm</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\cbind.scanoneperm.html" target=_blank>HTML\cbind.scanoneperm.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\c.scanone.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Combine columns from multiple scanone results</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlc_scanone</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\c.scanone.html" target=_blank>HTML\c.scanone.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\c.cross.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Combine data for QTL experiments</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlc_cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\c.cross.html" target=_blank>HTML\c.cross.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\c.scanoneperm.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Combine data from scanone permutations</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlc_scanoneperm</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\c.scanoneperm.html" target=_blank>HTML\c.scanoneperm.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\c.scantwoperm.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Combine data from scantwo permutations</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlc_scantwoperm</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\c.scantwoperm.html" target=_blank>HTML\c.scantwoperm.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\comparegeno.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Compare individuals' genotype data</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcomparegeno</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\comparegeno.html" target=_blank>HTML\comparegeno.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\ripple.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Compare marker orders</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlripple</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\ripple.html" target=_blank>HTML\ripple.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\comparecrosses.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Compare two cross objects.</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcomparecrosses</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\comparecrosses.html" target=_blank>HTML\comparecrosses.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\compareorder.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Compare two orderings of markers on a chromosome</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcompareorder</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\compareorder.html" target=_blank>HTML\compareorder.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\cim.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Composite interval mapping</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcim</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\cim.html" target=_blank>HTML\cim.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\condense.scantwo.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Condense the output from a 2-d genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcondense_scantwo</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\condense.scantwo.html" target=_blank>HTML\condense.scantwo.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\convert.scanone.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Convert output from scanone for R/qtl version 0.98</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlconvert_scanone</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\convert.scanone.html" target=_blank>HTML\convert.scanone.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\convert.scantwo.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Convert output from scantwo for R/qtl version 1.03 and earlier</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlconvert_scantwo</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\convert.scantwo.html" target=_blank>HTML\convert.scantwo.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\countXO.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Count number of obligate crossovers for each individual</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcountXO</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\countXO.html" target=_blank>HTML\countXO.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\geno.table.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Create table of genotype distributions</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlgeno_table</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\geno.table.html" target=_blank>HTML\geno.table.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\geno.crosstab.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Create table of two-locus genotypes</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlgeno_crosstab</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\geno.crosstab.html" target=_blank>HTML\geno.crosstab.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\bristleX.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Data on bristle number in Drosophila</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlbristleX</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\bristleX.html" target=_blank>HTML\bristleX.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\bristle3.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Data on bristle number in Drosophila</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlbristle3</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\bristle3.html" target=_blank>HTML\bristle3.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\hyper.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Data on hypertension</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlhyper</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\hyper.html" target=_blank>HTML\hyper.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\listeria.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Data on Listeria monocytogenes susceptibility</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtllisteria</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\listeria.html" target=_blank>HTML\listeria.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\dropfromqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Drop a QTL from a qtl object</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtldropfromqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\dropfromqtl.html" target=_blank>HTML\dropfromqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\drop.markers.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Drop a set of markers</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtldrop_markers</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\drop.markers.html" target=_blank>HTML\drop.markers.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\drop.nullmarkers.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Drop markers without any genotype data</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtldrop_nullmarkers</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\drop.nullmarkers.html" target=_blank>HTML\drop.nullmarkers.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\est.map.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Estimate genetic maps</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlest_map</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\est.map.html" target=_blank>HTML\est.map.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\est.rf.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Estimate pairwise recombination fractions</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlest_rf</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\est.rf.html" target=_blank>HTML\est.rf.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\fill.geno.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Fill holes in genotype data</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfill_geno</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\fill.geno.html" target=_blank>HTML\fill.geno.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\find.pheno.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Find column number for a particular phenotype.</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfind_pheno</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\find.pheno.html" target=_blank>HTML\find.pheno.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\find.flanking.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Find flanking markers for a specified position</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfind_flanking</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\find.flanking.html" target=_blank>HTML\find.flanking.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\find.marker.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Find marker closest to a specified position</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfind_marker</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\find.marker.html" target=_blank>HTML\find.marker.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\find.markerpos.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Find position of a marker.</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfind_markerpos</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\find.markerpos.html" target=_blank>HTML\find.markerpos.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\find.pseudomarker.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Find the pseudomarker closest to a specified position</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfind_pseudomarker</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\find.pseudomarker.html" target=_blank>HTML\find.pseudomarker.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\fitqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Fit a multiple-QTL model</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfitqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\fitqtl.html" target=_blank>HTML\fitqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\fitstahl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Fit Stahl interference model</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfitstahl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\fitstahl.html" target=_blank>HTML\fitstahl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\scanqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: General QTL scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlscanqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\scanqtl.html" target=_blank>HTML\scanqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\scanone.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Genome scan with a single QTL model</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlscanone</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\scanone.html" target=_blank>HTML\scanone.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\calc.errorlod.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Identify likely genotyping errors</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcalc_errorlod</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\calc.errorlod.html" target=_blank>HTML\calc.errorlod.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\checkAlleles.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Identify markers with switched alleles</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlcheckAlleles</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\checkAlleles.html" target=_blank>HTML\checkAlleles.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\add.cim.covar.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Indicate marker covariates from composite interval mapping</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtladd_cim_covar</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\add.cim.covar.html" target=_blank>HTML\add.cim.covar.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\qtlversion.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Installed version of R/qtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlqtlversion</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\qtlversion.html" target=_blank>HTML\qtlversion.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\qtl-internal.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Internal qtl functions</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlqtl_internal</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\qtl-internal.html" target=_blank>HTML\qtl-internal.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\a.starting.point.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Introductory comments on R/qtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtla_starting_point</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\a.starting.point.html" target=_blank>HTML\a.starting.point.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\jittermap.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Jitter marker positions in a genetic map</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtljittermap</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\jittermap.html" target=_blank>HTML\jittermap.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\top.errorlod.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: List genotypes with large error LOD scores</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtltop_errorlod</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\top.errorlod.html" target=_blank>HTML\top.errorlod.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\lodint.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: LOD support interval</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtllodint</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\lodint.html" target=_blank>HTML\lodint.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.scanoneperm.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: LOD thresholds from scanone permutation results</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_scanoneperm</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.scanoneperm.html" target=_blank>HTML\summary.scanoneperm.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.scantwoperm.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: LOD thresholds from scantwo permutation results</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_scantwoperm</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.scantwoperm.html" target=_blank>HTML\summary.scantwoperm.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\makeqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Make a qtl object</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlmakeqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\makeqtl.html" target=_blank>HTML\makeqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\max.scanone.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Maximum peak in genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlmax_scanone</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\max.scanone.html" target=_blank>HTML\max.scanone.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\max.scantwo.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Maximum peak in two-dimensional genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlmax_scantwo</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\max.scantwo.html" target=_blank>HTML\max.scantwo.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\movemarker.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Move a marker to a new chromosome</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlmovemarker</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\movemarker.html" target=_blank>HTML\movemarker.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\nmissing.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Number of missing genotypes</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlnmissing</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\nmissing.html" target=_blank>HTML\nmissing.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plotLodProfile.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot 1-d LOD profiles for a multiple QTL model</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplotLodProfile</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plotLodProfile.html" target=_blank>HTML\plotLodProfile.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.pheno.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot a phenotype distribution</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_pheno</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.pheno.html" target=_blank>HTML\plot.pheno.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\effectscan.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot estimated QTL effects across the whole genome</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtleffectscan</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\effectscan.html" target=_blank>HTML\effectscan.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.map.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot genetic map</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_map</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.map.html" target=_blank>HTML\plot.map.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.errorlod.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot grid of error LOD values</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_errorlod</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.errorlod.html" target=_blank>HTML\plot.errorlod.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\geno.image.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot grid of genotype data</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlgeno_image</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\geno.image.html" target=_blank>HTML\geno.image.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.missing.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot grid of missing genotypes</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_missing</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.missing.html" target=_blank>HTML\plot.missing.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.scanone.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot LOD curves</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_scanone</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.scanone.html" target=_blank>HTML\plot.scanone.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.scantwo.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot LOD scores for a two-dimensional genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_scantwo</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.scantwo.html" target=_blank>HTML\plot.scantwo.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.geno.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot observed genotypes, flagging likely errors</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_geno</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.geno.html" target=_blank>HTML\plot.geno.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.scantwoperm.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot permutation results for a 2d, 2-QTL genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_scantwoperm</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.scantwoperm.html" target=_blank>HTML\plot.scantwoperm.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.scanoneperm.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot permutation results for a single-QTL genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_scanoneperm</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.scanoneperm.html" target=_blank>HTML\plot.scanoneperm.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\effectplot.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot phenotype means against genotypes at one or two markers.</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtleffectplot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\effectplot.html" target=_blank>HTML\effectplot.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.pxg.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot phenotypes versus marker genotypes.</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_pxg</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.pxg.html" target=_blank>HTML\plot.pxg.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.qtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot QTL locations</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_qtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.qtl.html" target=_blank>HTML\plot.qtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.rf.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot recombination fractions</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_rf</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.rf.html" target=_blank>HTML\plot.rf.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.scanoneboot.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot results of bootstrap for QTL position.</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_scanoneboot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.scanoneboot.html" target=_blank>HTML\plot.scanoneboot.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.info.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot the proportion of missing genotype information</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_info</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.info.html" target=_blank>HTML\plot.info.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\plot.cross.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Plot various features of a cross object</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlplot_cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\plot.cross.html" target=_blank>HTML\plot.cross.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.map.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Print summary of a genetic map</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_map</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.map.html" target=_blank>HTML\summary.map.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.qtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Print summary of a QTL object</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_qtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.qtl.html" target=_blank>HTML\summary.qtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.cross.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Print summary of QTL experiment</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.cross.html" target=_blank>HTML\summary.cross.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.ripple.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Print summary of ripple results</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_ripple</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.ripple.html" target=_blank>HTML\summary.ripple.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\pull.pheno.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Pull out phenotype data from a cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlpull_pheno</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\pull.pheno.html" target=_blank>HTML\pull.pheno.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\pull.map.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Pull out the genetic map from a cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlpull_map</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\pull.map.html" target=_blank>HTML\pull.map.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\pull.geno.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Pull out the genotype data from a cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlpull_geno</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\pull.geno.html" target=_blank>HTML\pull.geno.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\read.cross.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Read data for a QTL experiment</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlread_cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\read.cross.html" target=_blank>HTML\read.cross.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\argmax.geno.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Reconstruct underlying genotypes</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlargmax_geno</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\argmax.geno.html" target=_blank>HTML\argmax.geno.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\refineqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Refine the positions of QTL</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlrefineqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\refineqtl.html" target=_blank>HTML\refineqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\clean.cross.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Remove derived data</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlclean_cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\clean.cross.html" target=_blank>HTML\clean.cross.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\reorderqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Reorder the QTL in a qtl object.</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlreorderqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\reorderqtl.html" target=_blank>HTML\reorderqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\replaceqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Replace a QTL in a qtl object with a different position.</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlreplaceqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\replaceqtl.html" target=_blank>HTML\replaceqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\replace.map.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Replace the genetic map of a cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlreplace_map</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\replace.map.html" target=_blank>HTML\replace.map.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\addpair.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Scan for an additional pair of QTL in a multiple-QTL model</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtladdpair</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\addpair.html" target=_blank>HTML\addpair.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\addqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Scan for an additional QTL in a multiple-QTL model</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtladdqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\addqtl.html" target=_blank>HTML\addqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\sim.map.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Simulate a genetic map</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsim_map</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\sim.map.html" target=_blank>HTML\sim.map.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\sim.cross.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Simulate a QTL experiment</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsim_cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\sim.cross.html" target=_blank>HTML\sim.cross.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\sim.geno.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Simulate genotypes given observed marker data</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsim_geno</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\sim.geno.html" target=_blank>HTML\sim.geno.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\fake.4way.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Simulated data for a 4-way cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfake_4way</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\fake.4way.html" target=_blank>HTML\fake.4way.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\fake.bc.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Simulated data for a backcross</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfake_bc</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\fake.bc.html" target=_blank>HTML\fake.bc.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\fake.f2.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Simulated data for an F2 intercross</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlfake_f2</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\fake.f2.html" target=_blank>HTML\fake.f2.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\strip.partials.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Strip partially informative genotypes</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlstrip_partials</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\strip.partials.html" target=_blank>HTML\strip.partials.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\subset.cross.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Subsetting data for QTL experiment</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsubset_cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\subset.cross.html" target=_blank>HTML\subset.cross.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\subset.scantwo.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Subsetting the results of a 2-d genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsubset_scantwo</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\subset.scantwo.html" target=_blank>HTML\subset.scantwo.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\subset.scanone.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Subsetting the results of a genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsubset_scanone</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\subset.scanone.html" target=_blank>HTML\subset.scanone.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.scanone.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Summarize the results of a genome scans</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_scanone</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.scanone.html" target=_blank>HTML\summary.scanone.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.scantwo.old.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Summarize the results of a two-dimensional genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_scantwo_old</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.scantwo.old.html" target=_blank>HTML\summary.scantwo.old.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.scantwo.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Summarize the results of a two-dimensional genome scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_scantwo</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.scantwo.html" target=_blank>HTML\summary.scantwo.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\summary.fitqtl.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Summary of fit of qtl model</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlsummary_fitqtl</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\summary.fitqtl.html" target=_blank>HTML\summary.fitqtl.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\switch.order.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Switch the order of markers on a chromosome</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlswitch_order</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\switch.order.html" target=_blank>HTML\switch.order.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\tryallpositions.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Test all possible positions for a marker</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtltryallpositions</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\tryallpositions.html" target=_blank>HTML\tryallpositions.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\scantwo.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Two-dimensional genome scan with a two-QTL model</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlscantwo</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\scantwo.html" target=_blank>HTML\scantwo.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\write.cross.html">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> R: Write data for a QTL experiment to a file</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> rqtlwrite_cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\write.cross.html" target=_blank>HTML\write.cross.html</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="RF_Plot.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> RF Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> RF_Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\RF_Plot.htm" target=_blank>RF_Plot.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Simulate_Cross.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Simulate Cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Simulate_Cross</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Simulate_Cross.htm" target=_blank>Simulate_Cross.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Simulate_Genotype_Probabilities.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Simulate Genotype Probabilities</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Simulate_Genotype_Probabilities</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Simulate_Genotype_Probabilities.htm" target=_blank>Simulate_Genotype_Probabilities.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="HTML\Two_QTL_Genome_Scan.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Two QTL Genome Scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Two_QTL_Genome_Scan</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\HTML\Two_QTL_Genome_Scan.htm" target=_blank>HTML\Two_QTL_Genome_Scan.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Two_QTL_Genome_Scan_Summary.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Two QTL Genome Scan Summary</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Two_QTL_Genome_Scan_Summary</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Two_QTL_Genome_Scan_Summary.htm" target=_blank>Two_QTL_Genome_Scan_Summary.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
<a name="Two_QTL_Scan_Plot.htm">
<table border=0 width=100% cellpadding=4 cellspacing=1 bgcolor=#808080>
<tr><td bgcolor=#D7D7D7><b>Topic Title:</b> Two QTL Scan Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>MapID:</b> Two_QTL_Scan_Plot</td></tr>
<tr><td bgcolor=#FFFFFF><b>File Path:</b> <a href="C:\Documents and Settings\kss\Desktop\j-qtl-help\Two_QTL_Scan_Plot.htm" target=_blank>Two_QTL_Scan_Plot.htm</a></td></tr>
<tr><td bgcolor=#FFFFFF><b>Index Keywords:</b> </td></tr>
<tr><td bgcolor=#FFFFFF><b>Status:</b> Not Specified<br><b>Notes: </b></td></tr>
</table><br>
</BODY></HTML>
