<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><title>R: Composite interval mapping</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link rel="stylesheet" type="text/css" href="../../R.css">
</head><body>

<table width="100%" summary="page for cim {qtl}"><tr><td>cim {qtl}</td><td align="right">R Documentation</td></tr></table>
<h2>Composite interval mapping</h2>


<h3>Description</h3>

<p>
Composite interval mapping by a scheme from QTL Cartographer: 
forward selection at the markers (here, with filled-in genotype data)
to a fixed number, followed by interval mapping with the selected
markers as covariates, dropping marker covariates if they are within
some fixed window size of the location under test.
</p>


<h3>Usage</h3>

<pre>
cim(cross, pheno.col=1, n.marcovar=3, window=10,
    method=c("em", "imp", "hk", "ehk"),
    imp.method=c("imp", "argmax"), error.prob=0.0001,
    map.function=c("haldane", "kosambi", "c-v", "morgan"),
    n.perm)
</pre>


<h3>Arguments</h3>

<table summary="R argblock">
<tr valign="top"><td><code>cross</code></td>
<td>
An object of class <code>cross</code>. See
<code><a href="read.cross.html">read.cross</a></code> for details.</td></tr>
<tr valign="top"><td><code>pheno.col</code></td>
<td>
Column number in the phenotype matrix which should be
used as the phenotype.  One may also give a character string matching
a phenotype name.</td></tr>
<tr valign="top"><td><code>n.marcovar</code></td>
<td>
Number of marker covariates to use.</td></tr>
<tr valign="top"><td><code>window</code></td>
<td>
Window size, in cM.</td></tr>
<tr valign="top"><td><code>method</code></td>
<td>
Indicates whether to use the EM algorithm, 
imputation, Haley-Knott regression, or the extended Haley-Knott
method.</td></tr>
<tr valign="top"><td><code>imp.method</code></td>
<td>
Method used to impute any missing marker genotype
data.</td></tr>
<tr valign="top"><td><code>error.prob</code></td>
<td>
Genotyping error probability assumed when imputing
the missing marker genotype data.</td></tr>
<tr valign="top"><td><code>map.function</code></td>
<td>
Map function used when imputing the missing marker
genotype data.</td></tr>
<tr valign="top"><td><code>n.perm</code></td>
<td>
If specified, a permutation test is performed rather than
an analysis of the observed data.  This argument defines the number
of permutation replicates.</td></tr>
</table>

<h3>Details</h3>

<p>
We first use <code><a href="fill.geno.html">fill.geno</a></code> to impute any missing marker
genotype data, either via a simple random imputation or using the
Viterbi algorithm.
</p>
<p>
We then perform forward selection to a fixed number of markers.  These
will be used (again, with any missing data filled in) as covariates in
the subsequent genome scan.
</p>


<h3>Value</h3>

<p>
The function returns an object of the same form as the function
<code><a href="scanone.html">scanone</a></code>:
<br>
If <code>n.perm</code> is missing, the function returns the scan results as
a data.frame with three columns: chromosome, position, LOD score.
Attributes indicate the names and positions of the chosen marker
covariates.
<br>
If <code>n.perm</code> &gt; 0, the function results the results of a
permutation test: a vector giving the genome-wide maximum LOD score in
each of the permutations.</p>

<h3>Author(s)</h3>

<p>
Karl W Broman, <a href="mailto:kbroman@biostat.wisc.edu">kbroman@biostat.wisc.edu</a>
</p>


<h3>References</h3>

<p>
Jansen, R. C. (1993) Interval mapping of multiple quantitative trait
loci.  <EM>Genetics</EM>, <B>135</B>, 205&ndash;211.
</p>
<p>
Jansen, R. C. and Stam, P. (1994) High resolution of quantitative
traits into multiple loci via interval mapping.  <EM>Genetics</EM>,
<B>136</B>, 1447-1455.
</p>
<p>
Zeng, Z. B. (1993) Theoretical basis for separation of multiple linked
gene effects in mapping quantitative trait loci.
<EM>Proc. Natl. Acad. Sci. USA</EM>, <B>90</B>, 10972&ndash;10976.
</p>
<p>
Zeng, Z. B. (1994) Precision mapping of quantitative trait loci.
<EM>Genetics</EM>, <B>136</B>, 1457&ndash;1468.
</p>


<h3>See Also</h3>

<p>
<code><a href="add.cim.covar.html">add.cim.covar</a></code>, <code><a href="scanone.html">scanone</a></code>, 
<code><a href="summary.scanone.html">summary.scanone</a></code>, <code><a href="plot.scanone.html">plot.scanone</a></code>,
<code><a href="fill.geno.html">fill.geno</a></code>
</p>


<h3>Examples</h3>

<pre>
data(hyper)
hyper &lt;- calc.genoprob(hyper, step=2.5)


out &lt;- scanone(hyper)
out.cim &lt;- cim(hyper, n.marcovar=3)
plot(out, out.cim, chr=c(1,4,6,15), col=c("blue", "red"))

add.cim.covar(out.cim, chr=c(1,4,6,15))
</pre>



<hr><div align="center">[Package <em>qtl</em> version 1.08-56 <a href="00Index.html">Index]</a></div>

</body></html>
