<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><title>R: Plot estimated QTL effects across the whole genome</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link rel="stylesheet" type="text/css" href="../../R.css">
</head><body>

<table width="100%" summary="page for effectscan {qtl}"><tr><td>effectscan {qtl}</td><td align="right">R Documentation</td></tr></table>
<h2>Plot estimated QTL effects across the whole genome</h2>


<h3>Description</h3>

<p>
This function is used to plot the estimated QTL effects along
selected chromosomes. For a backcross, there will be only one line,
representing the additive effect. For an intercross, there will be two
lines, representing the additive and dominance effects.
</p>


<h3>Usage</h3>

<pre>
effectscan(cross, pheno.col=1, chr, get.se=FALSE, draw=TRUE,
           gap=25, ylim, mtick=c("line","triangle"),
           add.legend=TRUE, alternate.chrid=FALSE, ...)
</pre>


<h3>Arguments</h3>

<table summary="R argblock">
<tr valign="top"><td><code>cross</code></td>
<td>
An object of class <code>cross</code>.</td></tr>
<tr valign="top"><td><code>pheno.col</code></td>
<td>
Column number in the phenotype matrix which to be
drawn in the plot.  One may also give a character string matching
a phenotype name.</td></tr>
<tr valign="top"><td><code>chr</code></td>
<td>
Chromosome(s) to be drawn in the plot (optional).</td></tr>
<tr valign="top"><td><code>get.se</code></td>
<td>
If TRUE, estimated standard errors are calculated.</td></tr>
<tr valign="top"><td><code>draw</code></td>
<td>
If TRUE, draw the figure.</td></tr>
<tr valign="top"><td><code>gap</code></td>
<td>
Gap separating chromosomes (in cM).</td></tr>
<tr valign="top"><td><code>ylim</code></td>
<td>
Y-axis limits (optional).</td></tr>
<tr valign="top"><td><code>mtick</code></td>
<td>
Tick mark type for markers.</td></tr>
<tr valign="top"><td><code>add.legend</code></td>
<td>
If TRUE, add a legend.</td></tr>
<tr valign="top"><td><code>alternate.chrid</code></td>
<td>
If TRUE and more than one chromosome is
plotted, alternate the placement of chromosome 
axis labels, so that they may be more easily distinguished.</td></tr>
<tr valign="top"><td><code>...</code></td>
<td>
Passed to the function <code><a href="../../graphics/html/plot.html">plot</a></code> when it
is called.</td></tr>
</table>

<h3>Details</h3>

<p>
The results of <code><a href="sim.geno.html">sim.geno</a></code> are required for taking
account of missing genotype information.
</p>
<p>
For a backcross, the additive effect is estimated as the difference
between the phenotypic averages for heterozygotes and homozygotes.
</p>
<p>
For recombinant inbred lines, the additive effect is estimated as half
the difference between the phenotypic averages for the two homozygotes.
</p>
<p>
For an intercross, the additive and dominance effects are estimated
from linear regression on <i>a</i> and <i>d</i> with <i>a</i> = -1, 0, 1,
for the AA, AB and BB genotypes, respectively, and <i>d</i> = 0, 1, 0,
for the AA, AB and BB genotypes, respectively.
</p>
<p>
As usual, the X chromosome is a bit more complicated.  We estimate
separate additive effects for the two sexes, and for the two
directions within females.
</p>
<p>
There is a function <code><a href="plot.effectscan.html">plot.effectscan</a></code> that creates
the actual plot by calling <code>plot.scanone</code>.  In the case
<code>get.se=TRUE</code>, colored regions indicate <i>+/-</i> 1 SE.
</p>


<h3>Value</h3>

<p>
The results are returned silently, as an object of class
<code>"effectscan"</code>, which is the same as the form returned by the
function <code><a href="scanone.html">scanone</a></code>, though with estimated effects
where LOD scores might be.  That is, it is a data frame with the first
two columns being chromosome ID and position (in cM), and subsequent
columns being estimated effects, and (if <code>get.se=TRUE</code>) standard
errors.</p>

<h3>Author(s)</h3>

<p>
Karl W. Broman, <a href="mailto:kbroman@biostat.wisc.edu">kbroman@biostat.wisc.edu</a>
</p>


<h3>References</h3>

<p>
Sen, 'S. and Churchill, G. A. (2001) A statistical framework for quantitative
trait mapping.  <EM>Genetics</EM> <B>159</B>, 371&ndash;387.
</p>


<h3>See Also</h3>

<p>
<code><a href="effectplot.html">effectplot</a></code>, <code><a href="plot.pxg.html">plot.pxg</a></code>,
<code><a href="sim.geno.html">sim.geno</a></code>
</p>


<h3>Examples</h3>

<pre>
data(fake.f2)

fake.f2 &lt;- sim.geno(fake.f2, step=2.5, n.draws=16)

# allelic effect on whole genome
effectscan(fake.f2)

# on chromosome 13, include standard errors
effectscan(fake.f2, chr="13", mtick="triangle", get.se=TRUE)
</pre>



<hr><div align="center">[Package <em>qtl</em> version 1.08-56 <a href="00Index.html">Index]</a></div>

</body></html>
