<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><title>R: Plot the proportion of missing genotype information</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link rel="stylesheet" type="text/css" href="../../R.css">
</head><body>

<table width="100%" summary="page for plot.info {qtl}"><tr><td>plot.info {qtl}</td><td align="right">R Documentation</td></tr></table>
<h2>Plot the proportion of missing genotype information</h2>


<h3>Description</h3>

<p>
Plot a measure of the proportion of missing information in the
genotype data.
</p>


<h3>Usage</h3>

<pre>
plot.info(x, chr, method=c("both","entropy","variance"), step=1,
          off.end=0, error.prob=0.001,
          map.function=c("haldane","kosambi","c-f","morgan"),
          alternate.chrid=FALSE, ...)
</pre>


<h3>Arguments</h3>

<table summary="R argblock">
<tr valign="top"><td><code>x</code></td>
<td>
An object of class <code>cross</code>.  See
<code><a href="read.cross.html">read.cross</a></code> for details.</td></tr>
<tr valign="top"><td><code>chr</code></td>
<td>
Vector specifying the chromosomes to plot.</td></tr>
<tr valign="top"><td><code>method</code></td>
<td>
Indicates whether to plot the entropy version of the
information, the variance version, or both.</td></tr>
<tr valign="top"><td><code>step</code></td>
<td>
Maximum distance (in cM) between positions at which the
missing information is calculated, though for <code>step=0</code>,
it is are calculated only at the marker locations.</td></tr>
<tr valign="top"><td><code>off.end</code></td>
<td>
Distance (in cM) past the terminal markers on each
chromosome to which the genotype probability calculations will be
carried.</td></tr>
<tr valign="top"><td><code>error.prob</code></td>
<td>
Assumed genotyping error rate used in the calculation
of the penetrance Pr(observed genotype | true genotype).</td></tr>
<tr valign="top"><td><code>map.function</code></td>
<td>
Indicates whether to use the Haldane, Kosambi or
Carter-Falconer map function when converting genetic distances into
recombination fractions.</td></tr>
<tr valign="top"><td><code>alternate.chrid</code></td>
<td>
If TRUE and more than one chromosome is
plotted, alternate the placement of chromosome 
axis labels, so that they may be more easily distinguished.</td></tr>
<tr valign="top"><td><code>...</code></td>
<td>
Passed to <code><a href="plot.scanone.html">plot.scanone</a></code>.</td></tr>
</table>

<h3>Details</h3>

<p>
The entropy version of the missing information: for a single
individual at a single genomic position, we measure the missing
information as <i>H = sum p[g] log
p[g] / log n</i>, where <i>p[g]</i> is the probability of the
genotype <i>g</i>, and <i>n</i> is the number of possible genotypes,
defining <i>0 log 0 = 0</i>.  This takes values between 0
and 1, assuming the value 1 when the genotypes (given the marker data)
are equally likely and 0 when the genotypes are completely determined.
We calculate the missing information at a particular position as the
average of <i>H</i> across individuals.  For an intercross, we don't
scale by <i>log n</i> but by the entropy in the case of genotype
probabilities (1/4, 1/2, 1/4). 
</p>
<p>
The variance version of the missing information: we calculate the
average, across individuals, of the variance of the genotype
distribution (conditional on the observed marker data) at a particular
locus, and scale by the maximum such variance.
</p>
<p>
Calculations are done in C (for the sake of speed in the presence of
little thought about programming efficiency) and the plot is created
by a call to <code><a href="plot.scanone.html">plot.scanone</a></code>.
</p>
<p>
Note that <code><a href="summary.scanone.html">summary.scanone</a></code> may be used to display
the maximum missing information on each chromosome.
</p>


<h3>Value</h3>

<p>
An object with class <code>scanone</code>: a data.frame with columns the
chromosome IDs and cM positions followed by the entropy and/or
variance version of the missing information.</p>

<h3>Author(s)</h3>

<p>
Karl W Broman, <a href="mailto:kbroman@biostat.wisc.edu">kbroman@biostat.wisc.edu</a>
</p>


<h3>See Also</h3>

<p>
<code><a href="plot.scanone.html">plot.scanone</a></code>,
<code><a href="plot.missing.html">plot.missing</a></code>
</p>


<h3>Examples</h3>

<pre>
data(hyper)

plot.info(hyper,chr=c(1,4))

# save the results and view maximum missing info on each chr
info &lt;- plot.info(hyper)
summary(info)
</pre>



<hr><div align="center">[Package <em>qtl</em> version 1.08-56 <a href="00Index.html">Index]</a></div>

</body></html>
